### PR TITLE
fix some golint warnings & import order

### PR DESCRIPTION
**Pull request by fhs**
The auto-generated files are ignored for now.

### CODE REVIEW

File: blas.go [hunk 1]
```diff
@@ -117,21 +117,21 @@ const (
 	Diagonal                    // Diagonal elements of H are units.
 )
 
-// Type SrotmParams contains Givens transformation parameters returned
```
Comment (btracey) [inline blas.go]: Could we remove all of the "A"? Just "// SrotmParams ... "

**Comment by btracey** [comment]
Where do I get the most up to date version of go lint? My local copy doesn't return any errors, which is clearly false.

**Comment by fhs** [comment]
go get -u github.com/golang/lint/golint
cd $GOPATH/src/github.com/gonum/blas
golint ./...

**Comment by btracey** [comment]
Thanks. I guess mine was out of date. 

What's wrong with a leading k on a variable name?

**Comment by btracey** [comment]
Or rather, what is the common thing people do to make that a golint error?

**Comment by btracey** [comment]
LGTM when the "A" are removed.

**Comment by fhs** [comment]
It's apparently a "common styles from other languages that don't belong in Go." See:
https://github.com/golang/lint/blob/master/lint.go#L486

I'm not sure what language it's referring to. Maybe people use "k" to mean "constant".

File: blas.go (hunk 1 above)
Comment (fhs) [inline blas.go]: PTAL

**Comment by btracey** [comment]
For the curious http://stackoverflow.com/questions/500030/what-is-the-significance-of-starting-constants-with-k

**Comment by btracey** [comment]
Thanks for the fixes.

**Pull request merged**